sockeye/data_io.py

```diff
@@ -1080,6 +1112,33 @@ def parallel_iter(source_iters: Sequence[Iterable[Optional[Any]]], target_iterab
         "Different number of lines in source(s) and target iterables.")
 
 
+class FileListReader(Iterator):
+    """
+    Reads sequence samples from path provided in a file.
+
+    :param fname: File name containing a list of relative paths.
+    :param path: Path to read data from, which is prefixed to the relative paths of fname.
+    """
+
+    def __init__(self,
+                 fname: str,
+                 path: str) -> None:
+        self.fname = fname
+        self.path = path
+        self.fd = smart_open(fname)
+        self.count = 0
+
+    def __next__(self):
+        fname = self.fd.readline().split("\n")[0]
```

Comment:
Wouldn't it be better to use strip()?

Reply:
Not familiar with it. I will check it out